llvm/tools/llvm-gsymutil/Opts.td

```diff
@@ -18,6 +18,8 @@ defm convert :
      "Convert the specified file to the GSYM format.\nSupported files include ELF and mach-o files that will have their debug info (DWARF) and symbol table converted">;
 def merged_functions :
   FF<"merged-functions", "Encode merged function information for functions in debug info that have matching address ranges.\nWithout this option one function per unique address range will be emitted.">;
+defm callsites_from_yaml :
+  Eq<"callsites-from-yaml", "Load call site info from YAML file.">;
```

Comment:
rename to "callsites-yaml-file"? "callsites-from-yaml" sounds like a boolean option name